packages/wallet/src/ui/forms/Form.tsx

```diff
@@ -26,40 +22,25 @@ import { XCircle } from '@/components/icons/X'
  *      });
  *
  *      return (
- *            <Form form={form} onSubmit={(data) => {
```

Comment:
do we need the comments?

Reply:
This is the general usage example on how to use the `Form` component along with `useZodForm`.